gradle/docker-build.gradle

```diff
@@ -0,0 +1,55 @@
+// script to build/push docker images.
+// The script setup, opposite to most of other gradle plugins, is quite obfuscated,
+// hence requires dedicated `buildscript` block and special classes imports.
+// see https://github.com/bmuschko/gradle-docker-plugin#remote-api-plugin
+// http://bmuschko.com/blog/dockerized-spring-boot-app/
+// Also, we don't use custom createDockerfile task in favor of static `docker-build/Dockerfile`,
+// which could be reused, for instance, for local native docker builds.
+// This could be changed in the future if we really need custom runtime flexible Dockerfile managing
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.bmuschko:gradle-docker-plugin:$dockerPackageVersion"
+    }
+}
+
+apply plugin: com.bmuschko.gradle.docker.DockerRemoteApiPlugin
+
+apply from: "$rootDir/gradle/util.gradle"
+
+import com.bmuschko.gradle.docker.tasks.image.DockerBuildImage
+import com.bmuschko.gradle.docker.tasks.image.DockerPushImage
+
+// project specific settings:
+final dockerContextDir = file('.')
+final dockerfileFile = file('docker-build/Dockerfile')
```

Comment:
is `dockerfile` not enough 😄 ?

Reply:
no

Reply:
Sorry, I confused by your comment. what do you mean by context?

Reply:
ah, i thought you mean that i specify 2 properties (`dockerContextDir` and `dockerfileFile`) instead of single one.

> is dockerfile not enough 

then answer is still no, because `dockerfile` is already defined in `DockerBuildImage` class/scope. See https://github.com/commercetools/commercetools-paypal-plus-integration/pull/124/files/14064964c5cd490423465bd7954cc936e94bd1ce#diff-f860522d3d883a88a1265ae077ddf680R37